src/Billable.php

```diff
@@ -623,4 +623,20 @@ public static function setStripeKey($key)
     {
         static::$stripeKey = $key;
     }
+
+    /**
+     * Get the account balance/credit for the entity.
+     *
+     * @return null|string
+     */
+    public function credit()
+    {
+        $balance = $this->asStripeCustomer()->account_balance;
+
+        if ($balance) {
+            return Cashier::formatAmount(- $balance);
+        }
+
+        return null;
```

Comment:
I think this should return 0 instead of null if there is no balance.

Reply:
https://github.com/laravel/cashier/pull/455/commits/e38a3e81d1f9d37d8e294b61ddb1c8a9b8abca3f